packages/devtools-launchpad/.babelrc

```diff
@@ -10,7 +10,7 @@
 
     ["module-resolver", {
       "alias": {
-        "react-dom": "react-dom/dist/react-dom",
+        "react-dom": "react-dom/umd/react-dom.development",
```

Comment:
Right. I’m curious why. In the past people used to do this to speed up DEV rebuilds (since React had too many modules). But now CJS is also a single bundle so maybe it‘s unnecessary.